src/scripts/settings/appSettings.js

```diff
@@ -12,15 +12,15 @@ import events from 'events';
     }
 
     export function enableAutoLogin(val) {
-        if (val != null) {
+        if (val !== undefined || null) {
```

Comment:
`val !== undefined || null` ~ `val !== undefined || false` ~ `val !== undefined`

maybe `val !== undefined && val !== null`?

Reply:
Hm yeah, good point. I was trying to make it shorter but that's not ideal :p